cms/static/js/spec/video/transcripts/videolist_spec.js

```diff
@@ -8,7 +8,9 @@ define(
     ],
 function ($, _, Utils, VideoList, MetadataModel, AbstractEditor, sinon) {
     'use strict';
-    describe('CMS.Views.Metadata.VideoList', function () {
+    // TODO: reenable these specs, using AjaxHelpers and the correct pattern
+    // to avoid flakiness. See TE-970 for details.
+    xdescribe('CMS.Views.Metadata.VideoList', function () {
```

Comment:
I almost have it implemented, if you can give me a few hours (I have a lot of meetings this morning).